contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
I will add that besides the three select locations in Julia outlined above, I do not foresee regular users or package authors needing to ever know about or use the `Base. DARWIN_FRAMEWORK` boolean variable.